tests/console/suseconnect_scc.pm

```diff
@@ -37,12 +37,6 @@ sub run {
     script_retry("$cmd", retry => 5, delay => 60, timeout => 180);
     # Check available extenstions (only present in sle)
     assert_script_run q[SUSEConnect --list-extensions];
-    if (is_sle) {
-        # What has been activated by default
-        assert_script_run q[SUSEConnect --list-extensions | grep -e '\(Activated\)'];
```

Comment:
Would it make sense to at least keep this check? This way we know that there is at least 1 module activated..

Reply:
I was thinking about it, but why ? On line 26 is first done registration.
This will print that there is something registered ? Cool, we can see that without the grep.

Reply:
But I don't mind, I can leave it there.

Reply:
yeah, kind of obvious check :) 
Ok to remove it. Added other reviewers just in case.

Reply:
It actually helps to see what is activated by default. Maybe the below ones are questionable. Nevertheless, instead of removal, I would vote for skipping the last 2 checks in case you don't expect those modules to be present in your product.

Reply:
afir that might be there just because doesn't fit in the screen the whole list.

Reply:
Sure, but it's useless anyway. Registration is done just step before.
This is like over-explaining buttons in elevator.

Reply:
There it is.